pycbc/strain.py

```diff
@@ -354,8 +381,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single'):
                              pd_taper_window) )
         gate_data(strain, gate_params)
 
-
+    strain.injcutter = injcutter
```

Comment:
@duncan-brown sounds like you want different names for the cutting based on chirp time vs based on coarse match?  or just less jargony in general? 
I suggested 'inj-reduced-bank' as that's what it does (reduce the bank searched for injections)